### PR TITLE
Simple permutation algorithm

### PR DESCRIPTION
**Pull request by KrzysztofNawara**

### CODE REVIEW

File: solvers.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    ffs_per_step = 1
+
+    def log_solution(sol, score):
+        log("solution: {}, score: {} /smaller-better/".format(sol, score))
+
+    def calc_next_solution(current):
+        G.reset_metadata()
+        new_solution = list(current)
+        random.shuffle(new_solution)
+        transitions, new_score = simulation(G, new_solution, init_nodes, ffs_per_step)[0:2]
```
Comment (piotroramus) [inline solvers.py:29]: simulation() in your code returns 3 values

**Review by piotroramus — CHANGES_REQUESTED** [review]
Nicely done. Just the one thing with simulation returning tuple should be changed.

File: solvers.py (hunk 1 above)
Comment (piotroramus) [reply solvers.py:29]: ok, nevermind, I did not see the [0:2]

Comment (KrzysztofNawara) [reply solvers.py:29]: But I'm taking it's subset ( [0:2] at the end )

**Pull request merged**